lib/remove-project.js

```diff
@@ -15,13 +19,20 @@ async function removeProject() {
     return;
   }
 
+  const allProjects = isUsingComponents
+    ? [{ id: "components", name: "Ditto Component Library" }, ...projects]
```

Comment:
On the backend of the public API, we still have some legacy code in project routes that treat the component library as a "special project" according to the identifier "ditto_component_library"; I think for consistency's sake it could be good to continue using that here, and hopefully eventually we can just completely move away from the pattern.

Reply:
sounds good pushed up a fix